documentation/Quickstart-Clear-Containers-in-a-Ubuntu-VM.md

```diff
@@ -0,0 +1,59 @@
+
+# Clear Containers Quick Start Development and Test environment
+
+## Introduction
+
+Clear Containers is normally run on bare metal machines with Intel CPU's
+supporting Intel VT-x. However Clear Containers can also be run inside a virtual
+machine provided the hypervisor support VT-x nesting. KVM supports VT-x nesting
+today.
+
+Installation of Containers on bare metal will result in changes to the host system
+and to the configuration of Docker on the host.
+
+In order to provide a seamless evaluation, development and test environment without
+the need for any modification to the developers host system, we provide an tools to
+create, launch and manage a fully configured virtual machine where you can try
+Clear Containers.
+
+This virtual machine is also setup with all the dependencies to build and modify the
+clear container runtime.
+
+This setup is particularly useful when your want try to modify networking functionality,
+or when trying out docker swarm or kubernetes.
+
+
+## ciao-down
+
+ciao-down is a small utility for setting up a VM that contains everything you need to run
+Clear Containers.
+
+All you need to have installed on your machine is:
+
+Install go 1.7 or greater on your host system, then
+
+```
+go get github.com/01org/ciao/testutil/ciao-down
+$GOPATH/bin/ciao-down prepare -vmtype clearcontainers
+$GOPATH/bin/ciao-down connect
+```
+Now you can launch clear containers within this virtual machine
+
+
+Now you can try clear containers inside the VM.
+
+For more details on ciao-down and extended usage instructions
+
+## Usage of ciao-down:
+
+ciao-down [prepare|start|stop|quit|status|connect|delete]
```

Comment:
I'd make use of backslashes to render this usage statement and the command names below in a fixed font for clarity.